### ISSUE TITLE
In usb-2-bot.md fehlen Ergänzungen zum neuen USB-Adapter

### ISSUE THREAD

**Issue by tsandmann** [Nightwalker-87/ct-bot-doku#20]
**Describe the bug**
In [usb-2-bot.md](https://github.com/Nightwalker-87/ct-bot-doku/blob/e0f96d3cda27e19dfa8da87529fbfa7009daf5f7/doc/wiki_pages_deprecated/usb-2-bot.md) fehlen die Ergänzungen von @r4dh4l zum neuen USB-Adapter.

**To Reproduce**
N/A

**Expected behavior**
Informationen zum neuen Adapter mit aufnehmen, wie z.B. [in der alten Version](https://github.com/tsandmann/ct-bot-doku/blob/69041f7df05e9552c329f992f2fc628117dd1b83/_tmp_trac_wiki_export/USB2Bot/USB2Bot.md#nachfolger).

**Desktop (please complete the following information):**
N/A

**Comment by Nightwalker-87** [comment]
Ja, das ist mir bekannt, diese werden auch wieder eingepflegt, allerdings in einem eigenen Artikel zu alternativen Adaptern, wo das thematisch auch passender ist.
